INSTALL.md

```diff
@@ -437,7 +424,7 @@ If your provider is able to allocate an IP Address from the Ingress controller t
 Unlike Openshift's `Route` the Kubernetes `Ingress` doesn't yet handle SSL termination. As such the default configuration will only expose AWX through HTTP on port 80. You are responsible for configuring SSL support until support is added (either to Kubernetes or AWX itself).
 
 
-## Docker-Compose
+## Community Docker Compose
```

Comment:
I think we should completely delete this section from this document. It's not a support install path. We should point people towards CONTRIBUTING.md.

Reply:
I see that you have a new README under tools/docker-compose. We could just point people at that.

Reply:
I removed those sections of the INSTALL.md.  I am still working on the best way to consolidate the `docker-compose/README.md` and `CONTRIBUTING.md`.